<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>227</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>351</width>
    <height>227</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>351</width>
    <height>227</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="userresources.qrc">
    <normaloff>:/Images/Images/icon.png</normaloff>:/Images/Images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="question">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>71</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>-1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Tahoma'; font-size:9pt;&quot;&gt;Haven't created an account yet?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SignUpShowForm">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>71</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Up!</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>99</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {font-family:Tahoma; font-size:10pt;font-weight: bold;}</string>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_Login_Password">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>59</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Login_Username">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>29</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>29</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="midLineWidth">
      <number>-1</number>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Tahoma'; font-size:10pt;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>59</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="midLineWidth">
      <number>-1</number>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Tahoma'; font-size:10pt;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>94</y>
       <width>321</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Login">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Close_1">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>lineEdit_Login_Password</zorder>
    <zorder>lineEdit_Login_Username</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>groupBox_SignUp</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_SignUp">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>65</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {font-family:Tahoma; font-size:10pt;font-weight: bold;}</string>
    </property>
    <property name="title">
     <string>Sign Up</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_SignUpPassword">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>59</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_SignUp_Username">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>29</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>29</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="midLineWidth">
      <number>-1</number>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Tahoma'; font-size:10pt;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>59</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="midLineWidth">
      <number>-1</number>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Tahoma'; font-size:10pt;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>126</y>
       <width>321</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_SignUp">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string>Sign Up!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Close_2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEdit_SignUpPassword_2">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>89</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>85</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="midLineWidth">
      <number>-1</number>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-family:'Tahoma'; font-size:10pt;&quot;&gt;Password&lt;br/&gt;again&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>lineEdit_SignUpPassword</zorder>
    <zorder>lineEdit_SignUp_Username</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>layoutWidget_2</zorder>
    <zorder>lineEdit_SignUpPassword_2</zorder>
    <zorder>label_6</zorder>
    <zorder>groupBox</zorder>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>-1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Tahoma'; font-size:14pt; font-weight:600; font-style:italic; color:#0000ff;&quot;&gt;AA &lt;/span&gt;&lt;span style=&quot; font-family:'Tahoma'; font-size:12pt; color:#0000ff;&quot;&gt;messanger&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="TitleBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#TitleBackground { background-image:url(:/Images/Images/background.png); }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Images/background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="ErrMsg">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>65</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {font-family:Tahoma; font-size:10pt;font-weight: bold;}</string>
    </property>
    <property name="title">
     <string>Error</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QTextBrowser" name="ErrLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#ErrLog { font-family:'Tahoma'; font-size:9pt; border:0px; background-color:#f0f0f0;}</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_SignUp</zorder>
   <zorder>TitleBackground</zorder>
   <zorder>Title</zorder>
   <zorder>groupBox</zorder>
   <zorder>question</zorder>
   <zorder>pushButton_SignUpShowForm</zorder>
   <zorder>ErrMsg</zorder>
  </widget>
 </widget>
 <resources>
  <include location="userresources.qrc"/>
 </resources>
 <connections/>
</ui>
